dill/_dill.py

```diff
@@ -1905,6 +1889,7 @@ def save_function(pickler, obj):
             # recurse to get all globals referred to by obj
             from .detect import globalvars
             globs_copy = globalvars(obj, recurse=True, builtin=True)
+            globs_copy.setdefault('__builtins__', globals()['__builtins__'])
```

Comment:
I'm not sure what you are asking me... I didn't write the line you are referring to, @leogama did.

Reply:
@mmckerns, This is just a variation of https://github.com/uqfoundation/dill/blob/a16219edf06393e41b9294a27f0511b41280ad2b/dill/_dill.py#L770-L771

But I didn't understand the question either. By default, `globals()['__builtins__']` in a module is "_an alias for the dictionary of the `builtins` module itself_". It could be substituted by the `__builtin__` alias in `_dill` though.

Reply:
Ah, ok.  So, unless I'm forgetting something... no reason.  I believe `__builtin__` could have been used here without issue, as that should point to the same module object as the current code.